src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -37,12 +39,29 @@ object Run  {
     def toRunMap = params.map(p => p.name -> p.gcs).toMap.asJava
   }
 
+  private def dateTryToString(dateTry: Try[DateTime]): String = dateTry match {
+    case scala.util.Success(d: DateTime) => d.toString
+    case Failure(e: Throwable) => e.getClass.getName
+  }
+
   sealed trait RunStatus
   trait TerminalRunStatus extends RunStatus
-  final case class Initializing(created: DateTime) extends RunStatus
-  final case class Running(created: DateTime, started: DateTime) extends RunStatus
-  final case class Success(created: DateTime, started: DateTime, finished: DateTime) extends TerminalRunStatus
-  final case class Failed(created: DateTime, started: DateTime, finished: DateTime, errorCode: Int, errorMessage: String) extends TerminalRunStatus
+  final case class Initializing(created: Try[DateTime]) extends RunStatus {
+    override def toString = s"Initializing(created: ${dateTryToString(created)})"
+  }
+
+  final case class Running(created: Try[DateTime], started: Try[DateTime]) extends RunStatus {
```

Comment:
I was thinking about this a bit and I think this whole scheme is incorrect. 

We should try to confirm the following statement w/ the google folks but I suspect what's happening here is that once in a while when we poll for a state it's not properly filled in. I realized earlier that I had seen this once and it was exactly that, and I promptly forgot to do anything about it.

If that's the case then I'd suggest the following is the right path forward. If the state of the JES job is non-terminal the only reason we're storing the dates is for logging purposes anyways (and perhaps we should only be logging when states transition instead of every pulse). If we receive a malformed non-terminal state we might as well skip it (e.g. via an "OOPS_SOMETHING_WENT_WRONG" state which is non-terminal) and if it *is* a terminal state well that actually might be a real error.

Another possibility is that we simply don't need the dates in the non-terminal states anyways - that was mainly to give me something to look at during those 10-15 minute job executions way back when.